.github/workflows/main.yml

```diff
@@ -17,14 +17,14 @@ jobs:
     steps:
       - uses: actions/setup-go@v4
         with:
-          go-version: '1.19.7'
+          go-version: '1.20.7'
```

Comment:
Is this the same version they are using?

Reply:
ya, I think so: https://github.com/prometheus/alertmanager/blob/4f1f020fe5e695105a0bee4511feed159f416842/.circleci/config.yml#L21